index.html

```diff
@@ -224,6 +224,36 @@ <h3 class="connect-text">LET'S CONNECT</h3>
       </div>
     </div>
   </section>
+
+  <section class="contact-form">
+    <div class="contact-form-holder">
+      <h2 class="contact-title">Contact me</h2>
+
+      <p class="p-pargraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/xqkwonvv" class="form-contact" method="post">
+        <ul class="ul-form">
+          <li class="li-form">
+            <input type="text" placeholder="Emmanue Hagan" name="name" id="name_form" maxlength="30" required />
+          </li>
+          <li class="li-form">
+            <input type="email" placeholder="haganemmanuel@gmail.com" name="email" id="email_form" required />
```

Comment:
- [ ] it would be better if the `placeholder` attribute like the ones in the Figma design, so please try to change it as in the Figma design